index.js

```diff
@@ -126,6 +160,20 @@ const guessModeHandlers = Alexa.CreateStateHandler(states.GUESSMODE, {
 
 // These handlers are not bound to a state
 const guessAttemptHandlers = {
+  PickACard(cardChoices) {
```

Comment:
Why are these capitalized?  I think I get it but I'm just curious what that style distinction is based on exactly.